tests/acl/test_acl.py

```diff
@@ -257,7 +258,7 @@ def teardown_rules(self, dut, setup):
         dut.command('config acl update full {}'.format(remove_rules_dut_path))
 
     @pytest.fixture(scope='class', autouse=True)
-    def acl_rules(self, duthost, localhost, setup, acl_table):
+    def acl_rules(self, duthost, ansible_adhoc, setup, acl_table):
```

Comment:
Maybe will be better to use "testbed_devices" fixture instead of "localhost".
Then you can get the correct localhost like this:
localhost = testbed_devices["localhost"]

Reply:
Will do.